src/Arcus.Messaging.Pumps.ServiceBus/Arcus.Messaging.Pumps.ServiceBus.csproj

```diff
@@ -29,7 +29,6 @@
   <ItemGroup>
     <PackageReference Include="Arcus.Security.Providers.AzureKeyVault" Version="[2.0.0,3.0.0)" />
     <PackageReference Include="Azure.Identity" Version="1.12.0" />
```

Comment:
I think we should upgrade Azure.Identity to a more recent version.
There are quite often new versions released of this package, so I was wondering if we should define this dependency as `Version=1.*` ?

Reply:
I would opt for version ranges, as it shows limitations at compile time io at runtime.

Reply:
But this is a bit an out-of-scope discussions as it spans all used Microsoft dependencies.